scripts/kind.sh

```diff
@@ -14,14 +14,40 @@ source "$SCRIPTS_DIR/lib/logging.sh"
 setup_kind_cluster() {
     local __cluster_name=$1
     local __controller_namespace=$2
-    local kubeconfig_path="$ROOT_DIR/build/clusters/$__cluster_name/kubeconfig"
+    local kind_cluster_build_dir="$ROOT_DIR/build/clusters/$__cluster_name"
+    local kubeconfig_path="$kind_cluster_build_dir/kubeconfig"
+    local kubecontext="kind-$__cluster_name"
+    local kubecontext_path="$kind_cluster_build_dir/kubecontext"
 
     info_msg "Creating cluster with name \"$__cluster_name\""
     _create_kind_cluster "$__cluster_name" "$kubeconfig_path"
 
-    info_msg "Exporting KUBECONFIG=$kubeconfig_path"
-    export KUBECONFIG=$kubeconfig_path
-
+    info_msg "Creating kubecontext file $kubecontext_path ... "
```

Comment:
nit: use `debug_msg` for this and `info_msg` for the lines below rather than `echo`

Reply:
done.